hooks/pre_commit/42-evr_extension_permissions.rb

```diff
@@ -0,0 +1,38 @@
+# Managed databases will be handled automatically.
+return if local_postgresql?
```

Comment:
You can't return outside of a function and it broke nightly pipelines
```
[2024-11-20T22:10:35.810Z]     /usr/share/foreman-installer/hooks/pre_commit/42-evr_extension_permissions.rb:2:in `block (4 levels) in load': unexpected return (LocalJumpError)
[2024-11-20T22:10:35.810Z]             from /usr/share/gems/gems/kafo-7.6.0/lib/kafo/hooking.rb:36:in `instance_eval'
[2024-11-20T22:10:35.810Z]             from /usr/share/gems/gems/kafo-7.6.0/lib/kafo/hooking.rb:36:in `block (4 levels) in load'
[2024-11-20T22:10:35.810Z]             from /usr/share/gems/gems/kafo-7.6.0/lib/kafo/hook_context.rb:19:in `instance_eval'
[2024-11-20T22:10:35.810Z]             from /usr/share/gems/gems/kafo-7.6.0/lib/kafo/hook_context.rb:19:in `execute'
[2024-11-20T22:10:35.810Z]             from /usr/share/gems/gems/kafo-7.6.0/lib/kafo/hooking.rb:67:in `block in execute'
[2024-11-20T22:10:35.810Z]             from /usr/share/gems/gems/kafo-7.6.0/lib/kafo/hooking.rb:65:in `each'
[2024-11-20T22:10:35.810Z]             from /usr/share/gems/gems/kafo-7.6.0/lib/kafo/hooking.rb:65:in `execute'
[2024-11-20T22:10:35.810Z]             from /usr/share/gems/gems/kafo-7.6.0/lib/kafo/kafo_configure.rb:219:in `execute'
[2024-11-20T22:10:35.810Z]             from /usr/share/gems/gems/clamp-1.3.2/lib/clamp/command.rb:66:in `run'
[2024-11-20T22:10:35.810Z]             from /usr/share/gems/gems/kafo-7.6.0/lib/kafo/kafo_configure.rb:184:in `run'
[2024-11-20T22:10:35.810Z]             from /usr/share/gems/gems/clamp-1.3.2/lib/clamp/command.rb:140:in `run'
[2024-11-20T22:10:35.810Z]             from /usr/share/gems/gems/kafo-7.6.0/lib/kafo/kafo_configure.rb:54:in `run'
[2024-11-20T22:10:35.810Z]             from /sbin/foreman-installer:8:in `<main>'
```

Reply:
Maybe this will solve it: https://github.com/theforeman/foreman-installer/pull/999